src/components/.config.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * This file is used for config file for consoleLogger, it is
+ * getting called in ConsoleLogComponent. config variable stores
+ * the config of the file.
+ * @param {string} config:  store the config option when debugging:
+ *  INFO: print console.info() on the browser console windows
+ *  WARN: print console.warn() on the browser console windows
+ *  ERROR: print console.error() on the brower console windows
+ * the order of the config string does not matter
+ */
+const config: string= "INFO, WARN, ERROR";
```

Comment:
This should only be a single value. 

INFO means -> Info, Warnings, and Errors appear
WARN means -> Warnings and Errors appear
ERROR means -> Just errors appear

Also how do you expect people to be able to update this to see more logs in their application? Once it's built you can't edit this. This is also a hard coded value, so only devs can change it and it will get tracked by VCS.